extensions/npm/main.ts

```diff
@@ -2,59 +2,125 @@ import { red, green, yellow } from 'https://deno.land/std@0.150.0/fmt/colors.ts'
 import { PhylumApi } from 'phylum';
 
 class FileBackup {
-  readonly fileName: string
-  readonly fileContent: string | null
+  readonly fileName: string;
```

Comment:
No need to change this for now, but for the future we should consider applying consistent automated styling to extensions. Generally I recommend https://standardjs.com/ which _doesn't_ have semicolons.

Reply:
Not having semicolons in JavaScript seems extremely unconventional to me.

Reply:
It's been pretty commonplace for quite a few years now. `npm` itself doesn't use them, I have not been using them for the largest part of my career when writing JS, most JS developers I know also do not use them. Then again the point is not whether they're good or bad, but having a consistent and easy to enforce system, and `standard` has been pretty great at doing that. But we can keep this for another PR.